plugins/fundchannel.c

```diff
@@ -310,7 +310,7 @@ static struct command_result *fundchannel_start(struct command *cmd,
 
 	json_out_start(ret, NULL, '{');
 	json_out_addstr(ret, "id", node_id_to_hexstr(tmpctx, fr->id));
-	json_out_addstr(ret, "satoshi", fr->funding_str);
```

Comment:
This still needs to be here, just `if (deprecated_apis) json_out_addstr(ret, "satoshi", fr->funding_str);`